cpp/Platform.Hashing/Platform.Hashing.Other.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#ifndef PLATFORM_HASHING_PLATFORM_HASHING_OTHER_H
+#define PLATFORM_HASHING_PLATFORM_HASHING_OTHER_H
+
+namespace std
+{
+#ifdef __cpp_lib_concepts
+    template<Platform::Hashing::not_std_hashable T>
+    struct hash<T>
+    {
+        size_t operator()(const T& item) const noexcept
+        {
+            return Platform::Hashing::RawHash(item);
+        }
+    };
+#else
+
```

Comment:
What should we do here? No way to support it without concepts?

Reply:
It can be done the same using SFINAE

Reply:
Can you give an example?